ddl/column_type_change_test.go

```diff
@@ -102,8 +102,7 @@ func TestColumnTypeChangeBetweenInteger(t *testing.T) {
 	tk.MustGetErrCode("alter table t modify column a mediumint", errno.ErrDataOutOfRange)
 	tk.MustGetErrCode("alter table t modify column a smallint", errno.ErrDataOutOfRange)
 	tk.MustGetErrCode("alter table t modify column a tinyint", errno.ErrDataOutOfRange)
-	_, err = tk.Exec("admin check table t")
-	require.NoError(t, err)
+	tk.MustExec("admin check table t")
```

Comment:
cloud be done in master

Reply:
I will create a PR to merge into master.

Reply:
after marging #35341 , I will revert the [ea199dc](https://github.com/pingcap/tidb/pull/32169/commits/ea199dca1ca23d923f5563b1b61f470535ffa042) to keep the cache unchangement.